docs/scos/dev/technical-enhancement-integration-guides/integrating-category-cms-blocks.md

```diff
@@ -17,295 +17,184 @@ related:
+5. Register the category subform and category relation read plugins.
 
-**\Pyz\Zed\Collector\CollectorDependencyProvider**
+**src/Pyz/Zed/CategoryGui/CategoryGuiDependencyProvider.php**
 
 ```php
 <?php
-namespace Pyz\Zed\Collector;
-...
 
-class CollectorDependencyProvider extends SprykerCollectorDependencyProvider
-{
-	public function provideBusinessLayerDependencies(Container $container)
-	{
-		...
-
-		$container[self::STORAGE_PLUGINS] = function (Container $container) {
-			return [
-				...
-				CmsBlockCategoryConnectorConstants::RESOURCE_TYPE_CMS_BLOCK_CATEGORY_CONNECTOR => new CmsBlockCategoryConnectorCollectorPlugin(),
-			];
-		};
-	}
-}
-```
-
-12. Register the Category form and form handler plugins.
-
-```php
-<?php
-namespace Pyz\Zed\CmsBlockGui;
+namespace Pyz\Zed\CategoryGui;
 
-...
+use Spryker\Zed\CategoryGui\CategoryGuiDependencyProvider as SpykerCategoryGuiDependencyProvider;
+use Spryker\Zed\CmsBlockCategoryConnector\Communication\Plugin\CategoryGui\CmsBlockCategoryRelationReadPlugin;
+use Spryker\Zed\CmsBlockCategoryConnector\Communication\Plugin\CategoryGui\CmsBlockSubformCategoryFormPlugin;
 
-class CategoryDependencyProvider extends SprykerDependencyProvider
+/**
+ * @method \Spryker\Zed\CategoryGui\CategoryGuiConfig getConfig()
+ */
+class CategoryGuiDependencyProvider extends SpykerCategoryGuiDependencyProvider
 {
-	...
-	/**
-     * @return \Spryker\Zed\Category\Dependency\Plugin\CategoryRelationUpdatePluginInterface[]
+    /**
+     * @return array<\Spryker\Zed\CategoryGuiExtension\Dependency\Plugin\CategoryFormPluginInterface>
      */
-    protected function getRelationUpdatePluginStack()
+    protected function getCategoryFormPlugins(): array
     {
-        return array_merge(
-            [
-				...
-                new CategoryFormPlugin(),
-            ],
-            parent::getRelationUpdatePluginStack()
-        );
+        return [
+            new CmsBlockSubformCategoryFormPlugin(),
+        ];
     }
 
-	/**
-     * @return array
+    /**
+     * @return array<\Spryker\Zed\CategoryGuiExtension\Dependency\Plugin\CategoryRelationReadPluginInterface>
      */
-    protected function getCategoryFormPlugins()
+    protected function getCategoryRelationReadPlugins(): array
     {
-        return array_merge(parent::getCategoryFormPlugins(), [
-			...
-            new CategoryFormPlugin()
-        ]);
+        return [
+            new CmsBlockCategoryRelationReadPlugin(),
+        ];
     }
 }
 ```
 
-13. Optional: To show which categories are assigned to a block on a block view page, register the category list plugin by adding `CmsBlockCategoryListViewPlugin` to the CMS Block GUI dependency provider:
+6. Optional: To show which categories are assigned to a block on a block view page, register the category list plugin by adding `CmsBlockCategoryListViewPlugin` to the CMS Block GUI dependency provider:
```

Comment:
Is that in the Back Office > Content > Blocks > View ?

Reply:
Yes